src/app/business/services/dynamic-form/dynamic-form.component.ts

```diff
@@ -160,6 +163,11 @@ export class DynamicFormComponent implements OnInit {
         
       })
       this.form = new FormGroup(formGroup);
+      if(this.selectedService.group == 'bigdata-analysis'){
```

Comment:
For this service we can do this hard check. But we will have to find a more scalable way to display the appropriate forms for custom workflows.
This can be added as an issue and backlog tag.